ariadne/src/Ariadne/Config/Cardano.hs

```diff
@@ -1,29 +1,333 @@
-import Pos.Launcher
+import Pos.Launcher (BaseParams(..), Configuration (..), LoggingParams(..))
+import Pos.Ssc.Configuration (SscConfiguration (..))
+import Pos.Ssc.Types (SscParams(..))
+import Pos.Txp.Configuration (TxpConfiguration(..))
+import Pos.Update.Params (UpdateParams(..))
+import Pos.Update.Configuration (UpdateConfiguration(..))
+import Pos.Util.UserSecret (UserSecret, peekUserSecret, usPrimKey, usVss, writeUserSecret)
+import Pos.Util.Util (eitherToThrow)
+import System.Wlog (HandlerWrap(..), LoggerConfig(..), LoggerTree(..), RotationParameters(..), WithLogger, logInfo)
+import System.Wlog.LoggerName (LoggerName)
+import System.Wlog.Severity (debugPlus)
+import Text.Show (Show(show))
+
+import qualified Pos.Launcher as Cardano (ConfigurationOptions(..), NodeParams(..))
+import qualified Pos.Client.CLI.NodeOptions as Cardano (CommonNodeArgs(..))
+import qualified Pos.Client.CLI.Options as Cardano (CommonArgs(..))
+import qualified Pos.Infra.Network.CLI as Cardano (NetworkConfigOpts(..))
+
+instance Default NtpConfiguration where
+    def = NtpConfiguration
+        { ntpcServers = ["0.pool.ntp.org", "2.pool.ntp.org", "3.pool.ntp.org"]
+        , ntpcResponseTimeout = 30000000
+        , ntpcPollDelay = 1800000000
+        }
+
+instance Default ApplicationName where
+    def = ApplicationName "cardano-sl"
+
+instance Default BlockVersion where
+    def = BlockVersion
+        { bvMajor = 0
+        , bvMinor = 0
+        , bvAlt = 0
+        }
+
+instance Default UpdateConfiguration where
+    def = UpdateConfiguration
+        { ccApplicationName = def
+        , ccLastKnownBlockVersion = def
+        , ccApplicationVersion = 0
+        , ccSystemTag = SystemTag ""
+        }
+
+instance Default SscConfiguration where
+    def = SscConfiguration
+        { ccMpcSendInterval = 10
+        , ccMdNoCommitmentsEpochThreshold = 3
+        , ccNoReportNoSecretsForEpoch1 = False
+        }
+
+instance Default DlgConfiguration where
+    def = DlgConfiguration
+        { ccDlgCacheParam = 500
+        , ccMessageCacheTimeout = 30
+        }
+
+instance Default TxpConfiguration where
+    def = TxpConfiguration { ccMemPoolLimitTx = 200 }
+
+instance Default BlockConfiguration where
+    def = BlockConfiguration
+        { ccNetworkDiameter = 3
+        , ccRecoveryHeadersMessage = 20
+        , ccNonCriticalCQBootstrap = 0.95
+        , ccCriticalCQBootstrap = 0.8888
+        , ccNonCriticalCQ = 0.8
+        , ccCriticalCQ = 0.654321
+        , ccCriticalForkThreshold = 2
+        , ccFixedTimeCQ = 10
+        }
+
+instance Default NodeConfiguration where
+    def = NodeConfiguration
+        { ccNetworkConnectionTimeout = 15000
+        , ccConversationEstablishTimeout = 30000
+        , ccBlockRetrievalQueueSize = 100
+        , ccPendingTxResubmissionPeriod = 7
+        , ccWalletProductionApi = False
+        , ccWalletTxCreationDisabled = False
+        }
+
+instance Default Configuration where
```

Comment:
No, this is wrong, this configuration won't work with mainnet. Mainnet configuration is very big, putting it into source code won't work. The idea is to embed configuration file (using `file-embed` probably) and parse it.